Formula/mqttx-cli.rb

```diff
@@ -0,0 +1,16 @@
+class MqttxCli < Formula
+  desc "MQTT 5.0 and MQTT X CLI client"
+  homepage "https://mqttx.app"
+  url "https://github.com/emqx/MQTTX/releases/download/v1.8.0/mqttx-cli.tar.gz"
```

Comment:
This ships a pre-built binary:
```
❯ curl -sL https://github.com/emqx/MQTTX/releases/download/v1.8.0/mqttx-cli.tar.gz | tar x > mqttx && file ./mqttx
./mqttx: Mach-O 64-bit x86_64 executable, flags:<NOUNDEFS|DYLDLINK|TWOLEVEL|WEAK_DEFINES|BINDS_TO_WEAK|HAS_TLV_DESCRIPTORS>
```
Formulae in `homebrew-core` must be built from source. See https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae.